tests/test-cases/Group1-Docker-Commands/1-17-Docker-Network-Connect.robot

```diff
@@ -99,7 +99,7 @@ Connect containers to multiple networks non-overlapping
 
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run --net cross2-network2 --name cross2-container2 debian ping -c2 ${ip}
     Should Not Be Equal As Integers  ${rc}  0
-    Should contain  ${output}  Destination Port Unreachable
+    Should Contain  ${output}  2 packets transmitted, 0 packets received, 100% packet loss
```

Comment:
We need two tests as you have pointed out before:
1. verify that bridge to management traffic is blocked
2. verify that bridge to management traffic is routed to the public network

For both these tests we need two port groups at least. Currently the CI only uses one port group, and I am not sure how to use another one. We most likely would add this to the nightly test run where the environment is more flexible. @rajanashok @mhagen-vmware